### ISSUE TITLE
Unexpected token: ,,

### ISSUE THREAD

**Issue by chipk4** [phpmd/phpmd#536]
(new StdClass)($text, $options)

**Comment by tinogo** [comment]
Yeah, would be awesome, if PHPMD could support invokable classes! :)

**Comment by PotterRafed** [comment]
Yes please, getting same errors for this exact issue

**Comment by koriym** [comment]
Would be nice!

**Comment by kylekatarnls** [comment]
This has been solved in PDepend 2.6.0.

Running `phpmd test.php text naming` with the following code:
```php
<?php

class MyClass
{
    public function myMethod()
    {
        (new StdClass)($a, $b);
    }
}
```

now properly output:
```
test.php:7     Avoid variables with short names like $a. Configured minimum length is 3.
test.php:7     Avoid variables with short names like $b. Configured minimum length is 3.
```
